docs/dev/setup/localci-localvc.rst

```diff
@@ -0,0 +1,100 @@
+^^^^^^^^^^^^^^^^^
+
+Modify ``src/main/resources/config/application-local.yml`` to include the correct URLs and directories:
+
+.. code:: yaml
+
+       artemis:
+           user-management:
+               use-external: false # if you do not wish to use Jira for user management
+           version-control:
+               url: http://localhost:8080
+
+.. HINT::
+   If you are running Artemis in Windows, you also need to add a property ``artemis.continuous-integration.docker-connection-uri`` with the value ``tcp://localhost:2375``.
+   The default value for this property is ``unix:///var/run/docker.sock`` which is defined in ``src/main/resources/config/application-localci.yml``. The ``application-local.yml`` file overrides the values set there.
+
+When you start Artemis for the first time, it will automatically create an admin user called "artemis_admin". If this does not work, refer to the guide for the :ref:`Jenkins and GitLab Setup` to manually create an admin user in the database.
+You can then use that admin user to create further users in Artemis' internal user management system.
+
+
+.. _Configure Jira:
+
+Configure Jira
+^^^^^^^^^^^^^^
+
+The local CI and local VC systems work fine without external user management configured so this step is **optional**.
+Setting up Jira allows you to run a script that sets up a number of users and groups for you.
+
+If you have already set up your system with Bamboo, Bitbucket, and Jira, you can keep using Jira for user management. Just stop the Bamboo and Bitbucket containers.
+If you want to use Jira for user management, but have not configured it yet, refer to the guide for the :ref:`Bamboo Bitbucket and Jira Setup`.
+You can follow all steps to set up the entire Atlassian stack, or just get the license for Jira and only follow steps 1-3 leaving out the setup of the Bamboo and Bitbucket containers.
+You can stop and remove the Bamboo and Bitbucket containers or just stop them in case you want to set them up later on.
+
+You also need to configure further settings in the ``src/main/resources/config/application-local.yml`` properties:
+
+.. code:: yaml
+
+       artemis:
+           user-management:
+               use-external: true
+               external:
+                   url: http://localhost:8081
+                   user:  <jira-admin-user> # insert the admin user you created in Jira
+                   password: <jira-admin-password> # insert the admin user's password
+                   admin-group-name: instructors
+
+
+.. _Start Artemis:
+
+Start Artemis
+^^^^^^^^^^^^^
+
+Start Artemis with the profiles ``localci`` and ``localvc`` so that the correct adapters will be used,
+e.g.:
+
+::
+
+   --spring.profiles.active=dev,localci,localvc,artemis,scheduling,local
+
+All of these profiles are enabled by default when using the ``Artemis (Server, LocalVC & LocalCI)`` run configuration in IntelliJ.
+Add ``jira`` to the list of profiles if you want to use Jira for user management.
```

Comment:
```suggestion
Add ``jira`` to the list of profiles if you want to use Jira for user management: `dev,localci,localvc,artemis,scheduling,local,jira`
```

Would this be overly specific? I think it could help unterstand this better in some cases.

Reply:
Yeah I think this is better.